modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImport/ModalImport.tsx

```diff
@@ -45,16 +50,23 @@ export default function ModalImport({
 	portletNamespace,
 	showModal,
 }: ModalImportProps) {
+	const [
+		alreadyImportedObjectDefinitions,
```

Comment:
Hey @danvb112, could you check with the BE guys what term is used in BE to refer to this situation?

Reply:
Hey @igor-franca, I talked with them and there is a pattern that they used because the call the serviceBuilder to do somethings, It's not our case, but I agree that this state name can be a little bit cofuse. Talking with Sá, he suggested use `existingObjectDefinitions` instead, I'll create a new commit renaming all the places that used `alreadyImportedObjectDefinitions` before